cli/build.rs

```diff
@@ -392,6 +392,20 @@ fn create_cli_snapshot(snapshot_path: PathBuf) -> CreateSnapshotOutput {
     deno_fs::deno_fs::init_ops::<PermissionsContainer>(fs.clone()),
     deno_node::deno_node::init_ops::<PermissionsContainer>(None, fs),
     deno_runtime::runtime::init_ops(),
+    deno_runtime::ops::runtime::deno_runtime::init_ops(
+      "deno:runtime".parse().unwrap(),
+    ),
+    deno_runtime::ops::worker_host::deno_worker_host::init_ops(
+      Arc::new(|_| panic!("not supported in this build.")),
```

Comment:
Maybe this to be clearer?

```suggestion
      Arc::new(|_| unreachable!("not used in snapshot")),
```